<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>202</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_BaseUrl">
     <property name="text">
      <string>Server URL:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_BaseUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="kcfg_BaseUrl">
     <property name="toolTip">
      <string>The URL of the Open-Xchange server, should be something like https://myserver.org/</string>
     </property>
     <property name="whatsThis">
      <string>The URL of the Open-Xchange server, should be something like https://myserver.org/</string>
     </property>
     <property name="placeholderText">
      <string notr="true">https://myserver.org</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Username">
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_Username</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="kcfg_Username">
     <property name="toolTip">
      <string>The username that is used to log into the Open-Xchange server</string>
     </property>
     <property name="whatsThis">
      <string>The username that is used to log into the Open-Xchange server</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Password">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_Password</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="kcfg_Password">
     <property name="toolTip">
      <string>The password that is used to log into the Open-Xchange server</string>
     </property>
     <property name="whatsThis">
      <string>The password that is used to log into the Open-Xchange server</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="checkConnectionButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Test Connection...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
